src/package.json

```diff
@@ -4,35 +4,25 @@
   "description": "Example of creating and deploying an API with Docker and Terraform on AWS",
   "author": "",
   "license": "Apache-2.0",
-  "main": "server.js",
+  "main": "index.js",
+  "type": "module",
   "scripts": {
     "lint": "standard --verbose | snazzy",
-    "test": "jest"
+    "test": "node --test server.test.js"
```

Comment:
```suggestion
    "test": "node --test --test-reporter=spec --test-reporter-destination=stdout --experimental-test-coverage --test-reporter=lcov --test-reporter-destination=lcov.info"
```
This is actually 4 suggestions in 1.
- Let's stop specifying `server.test.js` so that additional test files will likely be picked up automatically without needing to update this line. The [docs](https://nodejs.org/api/test.html#running-tests-from-the-command-line) state:
![image](https://github.com/byu-oit/hw-fargate-api/assets/7625337/1921dff0-73d8-4c86-90ef-290456d951de)
In my limited experience with the native test runner, tests inside of `node_modules` dependencies aren't picked up, so the default pattern matching should likely work for us.

- CI is [defaulting to `tap`](https://github.com/byu-oit/hw-fargate-api/actions/runs/8823142372/job/24222934833#step:5:12) results instead of the more human-readable `spec` ([due to some auto-detection](https://nodejs.org/api/test.html#test-reporters)). We can switch that by specifying `--test-reporter=spec`.
- We were previously summarizing test coverage data and reporting it at the end of the test results. We can do that by adding the `--experimental-test-coverage` flag [while using `tap` or `spec`](https://nodejs.org/api/test.html#coverage-reporters).
- We were previously sending test coverage details to Codecov [if a token was provided](https://github.com/byu-oit/hw-fargate-api/blob/5402ddec29b879c4f4f22b1a3989ef70b11fac41/.github/workflows/ci-app.yml#L41) (see [this](https://github.com/byu-oit/hw-fargate-api/pull/241)) by generating an LCOV report and placing it inside a `coverage` directory. The [Codecov action](https://github.com/codecov/codecov-action) recursively searches for `lcov.info`, ignoring directories like `node_modules`, and uploads it to report on diffs and track coverage trends. In order to generate that file _and_ print the human-readable stuff to stdout, we need to do something like `--test-reporter=spec --test-reporter-destination=stdout --experimental-test-coverage --test-reporter=lcov --test-reporter-destination=lcov.info`. A slightly awkward thing is that we can't specify `coverage/lcov.info` as the destination without the `coverage` directory already existing, so **we'll have to update our `.gitignore`**.